clients/src/main/resources/common/message/DescribeAclsRequest.json

```diff
@@ -22,11 +22,11 @@
   "validVersions": "0-2",
   "flexibleVersions": "2+",
   "fields": [
-    { "name": "ResourceType", "type": "int8", "versions": "0+",
+    { "name": "ResourceTypeFilter", "type": "int8", "versions": "0+",
```

Comment:
I would prefer to leave this as `ResourceType` if possible, since the Java class is `ResourceType.java` (there isn't a ResourceTypeFilter class or anything like that).